schema-next.yaml

```diff
@@ -2,6 +2,11 @@ file_format: 1.1.0
 schema_url: https://opentelemetry.io/schemas/next
 versions:
   next:
+    all:
+      changes:
+        - rename_attributes:
+            attribute_map:
+              device.id: installation.id
```

Comment:
Is this correct? `device.id` is still there, and I assume could still be used. 

I think we should not define any transformation and just explain this change in the changelog

Reply:
Alright, I can see that it is confusing for some, will revert the changes to `schema-next.yaml`.